internal/ingress/controller/controller.go

```diff
@@ -531,6 +531,11 @@ func (n *NGINXController) getBackendServers(ingresses []*ingress.Ingress) ([]*in
 				addLoc := true
 				for _, loc := range server.Locations {
 					if loc.Path == nginxPath {
+						// Same path but diffrent types are allowed
+						if *loc.PathType != *path.PathType {
+							break
+						}
```

Comment:
Can you point me to a test covering this? I don't understand why we are breaking out of this here.

Reply:
I added a new e2e test case